README.md

```diff
@@ -59,6 +59,10 @@ secrets:
 - **key** is the named secret in the secret reference.
 - **disabled** is optional and used to disable setting this environment value. This can be used to override the behaviour of default chart secrets. 
 
+## Postgresql
+
+Starting from version 2.0.0, Postgresql has been added to this chart as a dependency, but disabled by default.
```

Comment:
README's should reflect the current state, shouldn't have anything about previous version, the above sort of message is suited to release notes

How about something like:
```suggestion
If you need to use a Postgresql database for testing then you can enable it by setting the following flag:

java:
  postgresql:
  ## Whether to deploy the Postgres Chart or not
  enabled: true
  
You can then refer to it in your application config with:
java:
  environment:
    DB_HOST: "{{ .Release.Name }}-postgresql"
    DB_USER_NAME: "{{ .Values.postgresql.postgresqlUsername}}"
    DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
  
See the configuration section for more options if needed
```